pkg/aws/service.go

```diff
@@ -218,19 +218,19 @@ func FromEnv(ctx context.Context) Config {
 			IPNIDirectAnnounceURLs: ipniPublisherDirectAnnounceURLs,
 		},
 		ProvidersCacheExpirationSeconds:   mustGetInt("PROVIDERS_CACHE_EXPIRATION_SECONDS"),
-		NoProvidersCacheExpirationSeconds: mustGetInt("NO_PROVIDERS_CACHE_EXPIRATION_SECONDS"),
+		NoProvidersCacheExpirationSeconds: mustGetInt("NO-PROVIDERS_CACHE_EXPIRATION_SECONDS"),
 		ClaimsCacheExpirationSeconds:      mustGetInt("CLAIMS_CACHE_EXPIRATION_SECONDS"),
 		IndexesCacheExpirationSeconds:     mustGetInt("INDEXES_CACHE_EXPIRATION_SECONDS"),
-		SQSCachingQueueURL:                mustGetEnv("PROVIDER_CACHING_QUEUE_URL"),
-		CachingBucket:                     mustGetEnv("PROVIDER_CACHING_BUCKET_NAME"),
-		ChunkLinksTableName:               mustGetEnv("CHUNK_LINKS_TABLE_NAME"),
-		MetadataTableName:                 mustGetEnv("METADATA_TABLE_NAME"),
-		IPNIStoreBucket:                   mustGetEnv("IPNI_STORE_BUCKET_NAME"),
+		SQSCachingQueueID:                 mustGetEnv("PROVIDER-CACHING_QUEUE_ID"),
+		CachingBucket:                     mustGetEnv("PROVIDER-CACHING_BUCKET_NAME"),
```

Comment:
Ok, at this point I don't know if the dashes (instead of underscores) are intentional or not...so many.